src/models/Factor.js

```diff
@@ -158,7 +158,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
           // However, we currently have a problem with SMS
```

Comment:
can you update the comment two lines above to say ```Only push, sms and call have resend links.```

@gauthamgowrishankar-okta is SMS still broken for resend?